pkg/util/proxy_test.go

```diff
@@ -0,0 +1,145 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
```

Comment:
Wrong licence header and year. It should of Red Hat as `proxy` is of Red Hat and 2017 year